gamemodes/guesswho/gamemode/sh_config.lua

```diff
@@ -22,6 +22,10 @@ GM.Models = {
     "models/player/Group01/Female_03.mdl",
 }
 
+GM.PropFreezeEnabled = false -- if set to true, world props wont move.(default: false)
+GM.PropFreezeIsTimed = true -- if set false, props will be instantly freezed.(default: true, setting false may cause props stuck on air(EXPERIMENTAL))
```

Comment:
Probably remove these since they don't server any purpose? Why would you want to delay the freezing?